### PR TITLE
feat(whitelist): Add docs for specifying application whitelist using deis client

### PR DESCRIPTION
**Pull request by kmala**
Fixes https://github.com/deis/controller/issues/240

This should be merged after corresponding code changes are merged.

- [x] https://github.com/deis/controller-sdk-go/pull/87
- [x] https://github.com/deis/workflow-cli/pull/198
- [x] https://github.com/deis/controller/pull/1010

### CODE REVIEW

File: src/managing-workflow/security-considerations.md [hunk 1]
```diff
@@ -41,15 +41,23 @@ that hosts applications intended for a limited audience-- e.g. applications for
...
-$ kubectl --namespace=deis annotate service deis-controller router.deis.io/whitelist="10.0.1.0/24,121.212.121.212"
+    $ kubectl --namespace=deis annotate service deis-controller router.deis.io/whitelist="10.0.1.0/24,121.212.121.212"
 
 And the same applies to applications. For example, to apply a whitelist to an application named
 `example`:
 
-$ kubectl --namespace=example annotate service example-web router.deis.io/whitelist="10.0.1.0/24,121.212.121.212"
+    $ kubectl --namespace=example annotate service example-web router.deis.io/whitelist="10.0.1.0/24,121.212.121.212"
+
+Application level whitelist can also be done using deis client. To add/remove/list addresses of an application whitelist, use `deis whitelist`:
```
Comment (Joshua-Anderson) [inline src/managing-workflow/security-considerations.md]: > Application level whitelist can also be done using deis client.

Is a little awkward. Maybe "Application level **whitelists** can also be done using **the** deis client."

There also are not any example of listing application whitelists.

File: src/managing-workflow/security-considerations.md [hunk 2]
```diff
@@ -41,15 +41,27 @@ that hosts applications intended for a limited audience-- e.g. applications for
...
-$ kubectl --namespace=deis annotate service deis-controller router.deis.io/whitelist="10.0.1.0/24,121.212.121.212"
+    $ kubectl --namespace=deis annotate service deis-controller router.deis.io/whitelist="10.0.1.0/24,121.212.121.212"
 
 And the same applies to applications. For example, to apply a whitelist to an application named
 `example`:
 
-$ kubectl --namespace=example annotate service example-web router.deis.io/whitelist="10.0.1.0/24,121.212.121.212"
+    $ kubectl --namespace=example annotate service example-web router.deis.io/whitelist="10.0.1.0/24,121.212.121.212"
+
+Application level whitelisting can also be done using the deis client. To add/remove/list addresses of an application whitelist, use `deis whitelist`:
```
Comment (bacongobbler) [inline src/managing-workflow/security-considerations.md]: Deis should be capitalized

File: src/managing-workflow/security-considerations.md [hunk 3]
```diff
@@ -41,15 +41,27 @@ that hosts applications intended for a limited audience-- e.g. applications for
...
+
+    $ deis whitelist:add 10.0.1.0/24,121.212.121.212 -a drafty-zaniness
+		Adding 10.0.1.0/24,121.212.121.212 to drafty-zaniness whitelist...done
```
Comment (bacongobbler) [inline src/managing-workflow/security-considerations.md]: Extra whitespace here by accident?

File: src/managing-workflow/security-considerations.md [hunk 4]
```diff
@@ -41,15 +41,27 @@ that hosts applications intended for a limited audience-- e.g. applications for
...
+
+    $ deis whitelist:remove 121.212.121.212 -a drafty-zaniness
+    Removing 121.212.121.212 from drafty-zaniness whitelist... done
+
+		$ deis whitelist -a drafty-zaniness
```
Comment (bacongobbler) [inline src/managing-workflow/security-considerations.md]: extra whitespace here by accident?

Comment (kmala) [inline src/managing-workflow/security-considerations.md]: yeah..atom doesn't clearly show difference between tab and space.
changed it.

**Pull request merged**